config.py

```diff
@@ -24,6 +24,8 @@ class Config(object):
     JWT_BLACKLIST_ENABLED = True
     JWT_BLACKLIST_TOKEN_CHECKS = ["access", "refresh"]
     JWT_ACCESS_TOKEN_EXPIRES = 43200
+    ELASTICSEARCH_URL = os.environ.get("ELASTICSEARCH_URL")
```

Comment:
is this in the `.env` file somewhere? as manual export is not very ideal always
even better you could add a default 

```
ELASTICSEARCH_URL = os.environ.get('ELASTICSEARCH_URL', 'localhost:9200')
app.elasticsearch = Elasticsearch([app.config[ELASTICSEARCH_URL]])
```

Reply:
I Updated the config